Tavisca.Bootcamp.LanguageBasics.Exercise3/Program.cs

```diff
@@ -40,88 +40,79 @@ private static void Test(int[] protein, int[] carbs, int[] fat, string[] dietPla
 
         public static int[] SelectMeals(int[] protein, int[] carbs, int[] fat, string[] dietPlans)
         {
-            int menu_length       = protein.Length;
-            int[] total_calories  = new int[menu_length];
-            int[] selcted_menu    = new int[dietPlans.Length];
-            for (int i = 0; i < menu_length; i++){
-                total_calories[i] = (protein[i]*5) + (carbs[i] *5) + (fat[i]*9);
+            int menuLength       = protein.Length;
+            int[] totalCalories  = new int[menuLength];
+            int[] selctedMenu    = new int[dietPlans.Length];
+            for (int index = 0; index < menuLength; index++){
+                totalCalories[index] = (protein[index]*5) + (carbs[index] *5) + (fat[index]*9);
             }
-            int j = 0;
-            foreach (var item in dietPlans){
-                selcted_menu[j] = getMenuItem(item,protein,carbs,fat,total_calories);
-                j+=1;
+            for (int index = 0; index < dietPlans.Length; index++){
+                selctedMenu[index] = getMenuItem(dietPlans[index],protein,carbs,fat,totalCalories);
             }
-            return selcted_menu;
+            return selctedMenu;
         }
-        public static int getMenuItem(string item,int[] protein, int[] carbs, int[] fat,int[] total_calories){
-            List<int> temp_items = new List<int>();
-            for (int i = 0; i < protein.Length; i++){
-                temp_items.Add(i);
+        public static int getMenuItem(string dietPlan,int[] protein, int[] carbs, int[] fat,int[] totalCalories){

```

Comment:
Method name is not according to dotnet standards. It should start with a capital letter